pinot-core/src/test/java/com/linkedin/pinot/core/startree/HllBasicTest.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.core.startree;
+
+import com.clearspring.analytics.stream.cardinality.HyperLogLog;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+
+import java.util.Random;
+
+public class HllBasicTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(HllBasicTest.class);
+    private Random rand = new Random();
+
+    @Test
+    public void testExactHllFieldSize() throws Exception {
+        for (int i = 5; i < 15; i++) {
+            HyperLogLog hllValue = new HyperLogLog(i);
+            System.out.println("Log2m " + i + " empty size: " + hllValue.getBytes().length);
+            System.out.println("Estimated: " + hllValue.cardinality());
+            for (int j = 0; j < 100; j++) {
+                hllValue.offer(rand.nextLong());
+            }
+            System.out.println("Log2m " + i + " with 100 elements size: " + hllValue.getBytes().length);
+            System.out.println("Estimated: " + hllValue.cardinality());
+            for (int j = 0; j < 9900; j++) {
+                hllValue.offer(rand.nextLong());
+            }
+            System.out.println("Log2m " + i + " with 10000 elements size: " + hllValue.getBytes().length);
```

Comment:
add assertions here for automated testing. stdout writes are not good for verification